app/src/main/java/com/io/movio/domain/GetMoviesUseCase.kt

```diff
@@ -0,0 +1,12 @@
+package com.io.movio.domain
+
+import com.io.movio.data.models.Movie
+import com.io.movio.data.repositories.MovieRepository
+
+class GetMoviesUseCase(private val movieRepository: MovieRepository) {
+    operator fun invoke(): List<Movie> {
```

Comment:
you have two methods for the same thing. Why? And why do you need the operator keyword?